WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TextFieldStyles.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+/// Text field has a rounded border that has a thicker border and brighter border color when the field is focused.
+struct RoundedBorderTextFieldStyle: TextFieldStyle {
+    let focused: Bool
+
+    func _body(configuration: TextField<Self._Label>) -> some View {
+        configuration
+            .padding(10)
+            .background(
+                RoundedRectangle(cornerRadius: 8, style: .continuous)
+                    .stroke(focused ? Color(.brand): Color.gray,
```

Comment:
Should we make the border colors configurable with default values in case we want other colors in the future?

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/7995/commits/14df3c3ca025f5edc67b3a039f990f37110ad4da with some SwiftUI previews